modules/compiler/cookie/{{cookiecutter.target_name}}/source/module.cpp

```diff
@@ -285,20 +285,22 @@ void {{cookiecutter.target_name.capitalize()}}Module::initializePassMachineryFor
   llvm::Triple TT = llvm::Triple(target_machine->getTargetTriple());
   auto TLII = llvm::TargetLibraryInfoImpl(TT);
 
-  switch (CGO.getVecLib()) {
-    case clang::CodeGenOptions::Accelerate:
+  auto VecLib = CGO.getVecLib();
+  using VecLibT = decltype(VecLib);
```

Comment:
Actually, would it make sense to leave a comment here saying why we are doing this? Just in case someone sees this in the far future when LLVM 22 is out and can undo it.

Reply:
Sure, added a comment (in all three files that have this).